Dockerfile

```diff
@@ -3,20 +3,49 @@ FROM ${BUILDPLATFORM}alpine:3
 
 ARG KUBE_VERSION
 ARG HELM_VERSION
+ARG SOPS_VERSION
+ENV HELM_SECRETS_VERSION=3.12.0
```

Comment:
Let's have consistency here. If we have a previous ones as ARG what is the sense to put a new one as ENV?